nomad/structs/structs_test.go

```diff
@@ -1133,6 +1133,92 @@ func TestTaskGroup_Validate(t *testing.T) {
 
 }
 
+func TestTaskGroupNetwork_Validate(t *testing.T) {
+	cases := []struct {
+		TG          *TaskGroup
+		ErrContains string
+	}{
+		{
+			TG: &TaskGroup{
```

Comment:
Maybe add a positive non-erroring case as well?  Also, the differences between these conditions are somewhat subtle (Value vs To), and I thought the names were misspelling.  Not sure how to improve the names though...